.travis.yml

```diff
@@ -16,5 +16,7 @@ env:
 script:
   - if [[ $TEST_SUITE == "static" ]]; then flake8 && pydocstyle myia; fi
   - if [[ $TEST_SUITE == "unit" ]]; then pytest --cov=./ tests; fi
+  - if [[ $TEST_SUITE == "unit" ]]; then pytest tests; fi
+  - if [[ $TEST_SUITE == "unit" ]]; then pytest tests/test_grad.py::test_list_for; fi
```

Comment:
What are these for?

Reply:
Oh shit, those are leftovers from when I tried to figure out the problem.  I'll remove them.